plugins/services/src/js/components/forms/MultiContainerNetworkingFormSection.js

```diff
@@ -110,7 +110,7 @@ class MultiContainerNetworkingFormSection extends mixin(StoreMixin) {
       <span>
         {`This host port will be accessible as an environment variable called ${environmentVariableName}'. `}
         <a
-          href="https://mesosphere.github.io/marathon/docs/ports.html"
+          href="https://docs.mesosphere.com/1.11/deploying-services/service-ports/"
```

Comment:
🤔 @dcos/frontend Shouldn't use use a similar `buildMesosphereDocsURI()` for things like this, instead of hardcoding on the version in this string? Probably just one part of a bigger docs links refactor...

Reply:
@ahoskins yes we should - we already have a method for doing this. Can you please adjust this to use it?

Reply:
After confirming with Pavi, dcos.io now redirects to docs.mesosphere.com. So we will change our `buildDocsURI` method to this domain, and then can use it here.